src/core/public/application/ui/app_router.tsx

```diff
@@ -39,7 +39,7 @@ interface Props {
   redirectTo?: (path: string) => void;
 }
 
-export const AppRouter: React.StatelessComponent<Props> = ({
+export const AppRouter: React.FunctionComponent<Props> = ({
```

Comment:
`StatelessComponent` is deprecated due to react hooks. `FunctionComponent` is the new terminology.